CRVResponse/src/MakeCrvRecoPulses.cc

```diff
@@ -169,7 +176,8 @@ void MakeCrvRecoPulses::NoFitOption(const std::vector<unsigned int> &waveform, c
       {
         doublePulseThisPeak=false;
         doublePulseNextPeak=true;
-        sum=ADC; //the shared trough point gets added to both pulses
+        assert(i>0);
```

Comment:
please replace assert with an exception throw.

Reply:
Ok. I will change it.